.github/workflows/ci-cd.yml

```diff
@@ -64,10 +64,13 @@ jobs:
       codecov-token: ${{ matrix.upload-coverage && secrets.CODECOV_TOKEN || '' }}
 
   zizmor:
-    uses: ./.github/workflows/reusable-zizmor.yml
+    name: 🌈 zizmor
     permissions:
       security-events: write
 
+    # yamllint disable-line rule:line-length
+    uses: zizmorcore/workflow/.github/workflows/reusable-zizmor.yml@1ae473d8672fe7613e809d86d202a35063736e16
```

Comment:
There was an integration bug and this needs to be bumped to `3bb5e95068d0f44b6d2f3f7e91379bed1d2f96a8`: https://github.com/zizmorcore/workflow/issues/1.

Reply:
See https://github.com/aio-libs/aiobotocore/pull/1363